### PR TITLE
Bump NLTK version to 3.9 and revert to using nltk.download()

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

File: unstructured/nlp/tokenize.py [hunk 1]
```diff
@@ -16,132 +16,38 @@
...
-def check_for_nltk_package(package_name: str, package_category: str) -> bool:
-    """Checks to see if the specified NLTK package exists on the file system"""
-    paths: list[str] = []
-    for path in nltk.data.path:
-        if not path.endswith("nltk_data"):
-            path = os.path.join(path, "nltk_data")
-        paths.append(path)
 
+def _download_nltk_package_if_not_present(package_name: str, package_category: str):
+    """If the required nlt package is not present, download it."""
```
Comment (hteeyeoh) [inline unstructured/nlp/tokenize.py:26]: typo nlt

**Comment by cbornet** [comment]
@christinestraub is it still relevant or you want to continue using your custom downloader ?

**Comment by christinestraub** [comment]
Hi @cbornet,

Thank you for your valuable contribution and the time you've taken to submit this pull request. We truly appreciate your effort to improve our project.

I wanted to let you know that we've recently made some updates that affect this PR:

1. In our latest release (unstructured 0.15.7), we've already bumped the NLTK version to 3.9.1.
2. We're continuing to use our custom downloader for NLTK data, as it helps us address certain security vulnerabilities (CVEs).

Given these recent changes, we may not be able to merge this pull request as is. However, your attention to this issue is very helpful and we're grateful for your involvement in the project.

If you're interested in contributing further, please feel free to explore other areas of the project where your expertise could be valuable. We're always open to improvements and new ideas.

Thank you again for your contribution and for being part of our open-source community!

**Comment by cbornet** [comment]
OK. I close this one then.

**Pull request closed without merging**